lib/opentelemetry/trace/exporter/otlp.lua

```diff
@@ -96,12 +96,57 @@ function _M.export_spans(self, spans)
             }
         }
     }
+    local tracers = {}
```

Comment:
I think this is like `tracer_ils_idx_map`? 1:1 relationship between `tracer` and instrumentation library span index?

Reply:
this is the set of all tracers in the incomoing set of spans, and the value stored is the index of that span's il on the ils of the span's provider's resource